cmd/gaia/app/sim_test.go

```diff
@@ -161,12 +161,15 @@ func appStateFn(r *rand.Rand, accs []simulation.Account) json.RawMessage {
 	stakeGenesis.Validators = validators
 	stakeGenesis.Bonds = delegations
 
+	// TODO
```

Comment:
Is this still TODO? What needs to be done?

Reply:
It could be randomized.

Reply:
Now randomized.